HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs

```diff
@@ -12,34 +13,36 @@ public class RepairQueryHelper : IRepairQueryHelper
     {
         private readonly ContainerResponse cosmosContainer;
 
+        private readonly Expression<Func<Repair, bool>> isFutureRepair = x => x.Time.StartDateTime > DateTime.Today.Date;
+
         public RepairQueryHelper(ContainerResponse container) => cosmosContainer = container;
 
-        public FeedIterator<Repair> GetItemQueryIterator<T>(string repairType, string propertyReference) =>
-            cosmosContainer.Container.GetItemQueryIterator<Repair>(GetQueryDefinition(repairType, propertyReference));
+        public FeedIterator<Repair> GetRepairSearchIterator(string repairType, string propertyReference)
+        {
+            var query = cosmosContainer.Container.GetItemLinqQueryable<Repair>()
+                .Where(x =>
+                    x.RepairType.ToUpper() == repairType.ToUpper()
+                    && x.Address.LocationId == propertyReference)
+                .Where(isFutureRepair);
+
+            var result = query.ToFeedIterator();
```

Comment:
I'm just wondering if the conversion to using a feedIterator has preserved the date ordering functionality of the original SQL query:

`ORDER BY c.Time.StartDateTime ASC`

Reply:
And if this is needed or was a 'nice to have'.

Reply:
Restored and also added for tenant/leasehold queries